Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -88,7 +88,10 @@ void BtnCompareAPIClicked(object sender, RoutedEventArgs e)
             xApisSelectionGrid.btnMarkAll.Visibility = Visibility.Collapsed;
             xCompareBtnRow.Height = new GridLength(0);
 
-            xApisSelectionGrid.DataSourceList = AddAPIModelWizard.DeltaModelsList;
+            //Need to clear else if any item was selected on the Import Grid throws exception/error 
+            //as selected Item  won't anymore exist after updating the DataSource
+            AddAPIModelWizard.LearnedAPIModelsList.Clear();
+            xApisSelectionGrid.DataSourceList = new ObservableList<DeltaAPIModel>(AddAPIModelWizard.DeltaModelsList.OrderBy(d => d.comparisonStatus));
```

Comment:
but why you create new ObservableList<DeltaAPIModel> of Deta list before setting it as the grid DaraSource?

Reply:
We're _Ordering_ it based on the Comparison Status which returns _IOrderedEnumerable_ list of items while the DataSource is of type _IObservableList_